LanguageFeatures/Views/views_lib.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=extension-types
+
+library views_lib;
+
+int x = 42;
```

Comment:
Please add the newline at the end, you never know which (UNIX-ish) tools will be surprised when it's missing.